[2024-09-02T07:54:14.205+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-02T07:54:14.224+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_cron_expression_v3.task1 manual__2024-09-02T07:54:13.265662+00:00 [queued]>
[2024-09-02T07:54:14.233+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_cron_expression_v3.task1 manual__2024-09-02T07:54:13.265662+00:00 [queued]>
[2024-09-02T07:54:14.233+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-02T07:54:14.257+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): task1> on 2024-09-02 07:54:13.265662+00:00
[2024-09-02T07:54:14.265+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4717) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-02T07:54:14.265+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_cron_expression_v3', 'task1', 'manual__2024-09-02T07:54:13.265662+00:00', '--job-id', '445', '--raw', '--subdir', 'DAGS_FOLDER/cron_expression_dag.py', '--cfg-path', '/tmp/tmpbvr4fhie']
[2024-09-02T07:54:14.266+0000] {standard_task_runner.py:72} INFO - Started process 4722 to run task
[2024-09-02T07:54:14.267+0000] {standard_task_runner.py:105} INFO - Job 445: Subtask task1
[2024-09-02T07:54:14.313+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_cron_expression_v3.task1 manual__2024-09-02T07:54:13.265662+00:00 [running]> on host efc48c282cbe
[2024-09-02T07:54:14.411+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='dag_with_cron_expression_v3' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-02T07:54:13.265662+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-02T07:54:13.265662+00:00'
[2024-09-02T07:54:14.412+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-02T07:54:14.430+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-02T07:54:14.431+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo dag with cron expression!']
[2024-09-02T07:54:14.442+0000] {subprocess.py:86} INFO - Output:
[2024-09-02T07:54:14.443+0000] {subprocess.py:93} INFO - dag with cron expression!
[2024-09-02T07:54:14.444+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-02T07:54:14.472+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-02T07:54:14.473+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_cron_expression_v3, task_id=task1, run_id=manual__2024-09-02T07:54:13.265662+00:00, execution_date=20240902T075413, start_date=20240902T075414, end_date=20240902T075414
[2024-09-02T07:54:14.521+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-02T07:54:14.536+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-02T07:54:14.547+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
